maintenance/update-juju-solutions-fork.sh

```diff
@@ -6,9 +6,9 @@ set -eu
 # In the rare case that the merge fails, we can manually merge. The Jenkins job will notify via IRC when it fails.
 
 git checkout master
+git pull origin master
 if ! git remote|grep upstream; then
   git remote add upstream https://github.com/kubernetes/kubernetes.git
 fi
 git pull upstream master
-git merge upstream/master
```

Comment:
Opportunistic cleanup here - the `git pull upstream master` already handles the merge, we don't need to do it again.